pkg/scheduler/framework/plugins/apienablement/api_enablement.go

```diff
@@ -68,7 +68,8 @@ func (p *APIEnablement) Filter(
 	// we enforce strict checks to maintain consistency. This may occasionally
 	// exclude clusters prematurely. Users requiring a specific number of target
 	// clusters should use SpreadConstraints(in PropagationPolicy) to meet their requirements.
-	if helper.IsAPIEnabled(cluster.Status.APIEnablements, bindingSpec.Resource.APIVersion, bindingSpec.Resource.Kind) {
+	apiEnablement := cluster.APIEnablement(schema.FromAPIVersionAndKind(bindingSpec.Resource.APIVersion, bindingSpec.Resource.Kind))
+	if apiEnablement == clusterv1alpha1.APIEnabled {
```

Comment:
Does the logic on the scheduler side need to be modified synchronously: when it is Unknown, schedule.
@RainbowMango

Reply:
Why?

Reply:
No. This plugin is used to select candidates from a cluster pool, it should enforce strict checks. This is quite differenct from other cases.